docs/README.vlan.bridging.topo.md

```diff
@@ -0,0 +1,266 @@
+
+Create network namespace for end hosts and inject one end of veth pair
+
+```
+sudo ip netns add host-1.0
+sudo ip netns add host-2.0
+sudo ip netns add host-3.0
+sudo ip netns add host-4.0
+sudo ip netns add host-5.0
+sudo ip netns add host-6.0
+sudo ip link set dev veth_ac1 netns host-1.0
+sudo ip link set dev veth_ac2 netns host-2.0
+sudo ip link set dev veth_ac3 netns host-3.0
+sudo ip link set dev veth_ac4 netns host-4.0
+sudo ip link set dev veth_ac5 netns host-5.0
+sudo ip link set dev veth_ac6 netns host-6.0
+```
+
+Configure IP addresses inside the host net namepaces
+
+```
+sudo ip netns exec host-1.0 bash
+ip link set dev veth_ac1 up
+ip addr add 10.0.1.1/24 dev veth_ac1
+ip address show
+ip route show
+exit
+sudo ip netns exec host-2.0 bash
+ip link set dev veth_ac2 up
+ip addr add 10.0.2.1/24 dev veth_ac2
+ip address show
+ip route show
+exit
+sudo ip netns exec host-3.0 bash
+ip link set dev veth_ac3 up
+ip addr add 10.0.3.1/24 dev veth_ac3
+ip address show
+ip route show
+exit
+sudo ip netns exec host-4.0 bash
+ip link set dev veth_ac4 up
+ip addr add 10.0.1.2/24 dev veth_ac4
+ip address show
+ip route show
+exit
+sudo ip netns exec host-5.0 bash
+ip link set dev veth_ac5 up
+ip addr add 10.0.2.2/24 dev veth_ac5
+ip address show
+ip route show
+exit
+sudo ip netns exec host-6.0 bash
+ip link set dev veth_ac6 up
+ip addr add 10.0.3.2/24 dev veth_ac6
+ip address show
+ip route show
+exit
+```
+
+Now start the sonic container and pass the veth interfaces to sonic-vpp
+```
+sudo ./start_sonic_vpp.sh start -n sonic-br1 -i ac1,ac2,ac3,trunk
+```
+
+start the second sonic container and pass the veth interfaces to sonic-vpp
+```
+sudo ./start_sonic_vpp.sh start -n sonic-br2 -i ac3,ac4,ac5,veth_trunk
+```
+
+Get into sonic-br1 container and configure vlan and vlan bridging
+sonic-br1:
+```
+docker exec -it sonic-br2 /bin/bash
+show interface status
+config interface startup Ethernet0
+config interface startup Ethernet1
+config interface startup Ethernet2
+config interface startup Ethernet3
+
+config vlan add 10
+config vlan member add -u 10 Ethernet0
+config vlan member add  10 Ethernet3
+
+config vlan add 20
+config vlan member add -u 20 Ethernet1
+config vlan member add  20 Ethernet3
+
+config vlan add 30
+config vlan member add -u 30 Ethernet2
+config vlan member add  30 Ethernet3
+```
+
+Get into sonic-br2 container and configure vlan , vlan bridging
+sonic-br2:
+```
+docker exec -it sonic-br2 /bin/bash
+show interface status
+config interface startup Ethernet0
+config interface startup Ethernet1
+config interface startup Ethernet2
+config interface startup Ethernet3
+
+config vlan add 10
+config vlan member add -u 10 Ethernet0
+config vlan member add  10 Ethernet3
+
+config vlan add 20
+config vlan member add -u 20 Ethernet1
+config vlan member add  20 Ethernet3
+
+config vlan add 30
+config vlan member add -u 30 Ethernet2
+config vlan member add  30 Ethernet3
+```
+Test the SONiC vlan briding function across 3 VLAN bridge domains
```

Comment:
Do you want to ping test in other direction? Host 4,5,6 to 10.0.x.1?

Reply:
Yes It works. Will add other direction as well.